joomla/joomla-cms
# [4.0] Improve text in batch modal

Alternate to #24817.

### Summary of Changes
Make Copy/Move label consistent with other labels.


### Testing Instructions
Content > Articles > select some articles > click on Batch
Select a category so that the Copy / Move question is shown.

The radio buttons will be fixed if this is approved.

### Expected result
![batch](https://user-images.githubusercontent.com/368084/77458728-e5cdc080-6dbb-11ea-830a-1640a077960e.jpg)
